runbooks/mcp-to-permissioned-fps.md

```diff
@@ -370,9 +375,13 @@ You will need to set the following flags for `op-dispute-mon` at a minimum:
     - `--honest-actors, HONEST_ACTORS_FLAG`
     - Ensure that the address of the `Proposer` is included in this list
 
-# Executing the Finalization Transaction
+## Executing the finalization transaction
 
 > [!IMPORTANT]
 > Transaction bundles **MUST** be executed atomically within a single transaction. Do NOT execute each transaction within the bundle individually. Executing the bundle non-atomically is UNSAFE and can open the system up to critical vulnerabilities.
 
 You should have generated a JSON file called `bundle.json` when you deployed the required contracts. `bundle.json` contains a list of transactions compatible with the Transaction Builder feature for the Safe smart wallet. Once you execute this transaction bundle, your chain will be running the `op-contracts/v1.6.0` system. You should make sure that you are successfully able to run the services described above before executing this transaction bundle.
+
+## Updating the superchain-registry
+
+If your chain is in the [superchain-registry](https://github.com/ethereum-optimism/superchain-registry/tree/main), make sure to open a PR to update the chain information.
```

Comment:
What information needs to be updated? Would be nice if we can include a bit more info here

Reply:
Added some more information to fill the gap